### PR TITLE
Change message for Entrypoint type name error

### PR DESCRIPTION
**Pull request by PMoneda**
Guys I change to message when the EnpointEtry has no type called Program, I spent 20 minutes to discover that error in my program was the name of Entrypoint class.
The message that DNX show me was about my Main method but the main method was right and the error was in the name of class. So, i change de error message to be more specific in that case.

What do you think?

### CODE REVIEW

File: src/Microsoft.Dnx.Runtime.Sources/Impl/EntryPointExecutor.cs [hunk 1]
```diff
@@ -90,7 +90,8 @@ private static bool TryGetEntryPoint(Assembly assembly, IServiceProvider service
 
                 if (programTypeInfo == null)
                 {
-                    System.Console.WriteLine("'{0}' does not contain a static 'Main' method suitable for an entry point", name);
+                                        
+                    System.Console.WriteLine("'{0}' does not contain a type 'Program' for an entry point", name);
```
Comment (anurse) [inline src/Microsoft.Dnx.Runtime.Sources/Impl/EntryPointExecutor.cs]: Slight reword suggestion: '{0}' does not contain a 'Program' type suitable for an entry point

**Comment by anurse** [comment]
Minor reword suggestion but otherwise looks good to me!

File: src/Microsoft.Dnx.Runtime.Sources/Impl/EntryPointExecutor.cs [hunk 2]
```diff
@@ -81,7 +81,7 @@ private static bool TryGetEntryPoint(Assembly assembly, IServiceProvider service
             {
                 return true;
             }
-
+
```
Comment (moozzyk) [inline src/Microsoft.Dnx.Runtime.Sources/Impl/EntryPointExecutor.cs]: nit: revert

**Comment by PMoneda** [comment]
@anurse I changed the message to "'{0}' does not contains a 'Program' type suitable for an entry point"

**Comment by anurse** [comment]
Looks good to me! Clean up the extra space added (@moozzyk 's comment) and I'll merge it!

**Comment by PMoneda** [comment]
@anurse, @moozzyk 's comment it's done!

**Comment by anurse** [comment]
Looks good! Thanks for the contribution!

**Pull request merged**